inspirehep/modules/search/__init__.py

```diff
@@ -27,5 +27,8 @@
 from __future__ import absolute_import, print_function
 
 from .ext import INSPIRESearch
+from .query_factory import inspire_query_factory
 
-__all__ = ('INSPIRESearch', )
+IQ = inspire_query_factory()
```

Comment:
I took it as inspiration from https://github.com/inveniosoftware/invenio-query-parser/blob/master/invenio_query_parser/contrib/elasticsearch/__init__.py but yeah, might be better to do a Proxy.